### ISSUE TITLE
dired-filter:  let: Symbol’s value as variable is void: file-name

### ISSUE THREAD

**Issue by kermitwk** [Fuco1/dired-hacks#112]
I update the latest `dired-filter` from `melpa`,  and when I press `C-x d` to get into a directory,  I get the following error:

> let: Symbol’s value as variable is void: file-name

Here is my configuration:

```elisp
(require 'dired-filter)
(add-hook 'dired-mode-hook 'dired-filter-group-mode)
(define-key dired-mode-map (kbd "/") dired-filter-map)
(setq dired-filter-group-saved-groups
      '(("default"
         ("elisp"
          (extension . "el"))
         ("python"
          (extension . "py"))
         ("sh"
          (extension . "sh"))
         ("pdf"
          (extension . "pdf"))
         ("LaTeX"
          (extension "tex" "bib"))
         ("org"
          (extension . "org"))
         ("markdown"
          (extension . "md"))
         ("archives"
          (extension "zip" "rar" "gz" "bz2" "tar")))))
```

**Comment by Fuco1** [comment]
Can you `M-x toggle-debug-on-error` and make sure it is enabled, then run the command again and past the entire stack trace?

**Comment by Fuco1** [comment]
Nevermind, after a restart this is happening to me as well.

**Comment by Fuco1** [comment]
Should be fixed by 64b3dbb.
